language/de/acp/board.php

```diff
@@ -409,6 +409,8 @@
 	'RECOMPILE_STYLES_EXPLAIN'		=> 'Prüft auf neue Style-Komponenten und rekompiliert diese.',
 	'YES_ANON_READ_MARKING'			=> 'Gelesen-Markierung für Gäste',
 	'YES_ANON_READ_MARKING_EXPLAIN'	=> 'Speichert auch für Gäste, ob ein Thema gelesen oder ungelesen ist. Wenn diese Option deaktiviert ist, erscheinen Beiträge für Gäste immer als gelesen.',
+	'YES_ACCURATE_PM_BUTTON'			=> 'Aktiviert den berechtigungsabhängigen PN-Button in der Themenansicht',
```

Comment:
Haben wir sonst irgendwo Button verwendet. Sonst: Schaltfläche

Reply:
Ne, soweit ich das sehe haben wir nirgends "Button". In "'FIELD_IS_CONTACT_EXPLAIN'" sind es Kontaktfelder.

Reply:
Das wäre dann der einzige verbliebene Punkt, oder?